www/grid_view/src/module/grid.controller.coffee

```diff
@@ -28,6 +28,7 @@ class Grid extends Controller
         @changeFetchLimit = settings.changeFetchLimit.value
         @buildFetchLimit = settings.buildFetchLimit.value
         @compactChanges = settings.compactChanges.value
+        @rightToLeft = settings.rightToLeft.value
 
         @buildsets = @data.getBuildsets(
             limit: @changeFetchLimit
```

Comment:
Here you should rather take @buildFetchLimit. If the trigger step is used, the number of buildset is more on the order of the number of build rather than the order of change.
better to many than too few.